context.ts

```
@@ -173,7 +179,13 @@ function createPluginLogger(name: string): winston.Logger {
   });
 }
 
-const context: Context = {
+function warnServerFunc(functionName: string) {
+  console.error(
```

I wanted to throw a new Error, but the exception seems to be swallowed and never reaches the test. Do you know why?

---

Was a problem on my side... Updated to throw...